build/tasks/server.js

```diff
@@ -11,7 +11,9 @@ app.all('*', function(req, res) {
     path: req.path,
     query: req.query,
     body: req.body,
-    method: req.method
+    method: req.method,
+    contentType: req.header('content-type'),
+    Authorization: req.header('Authorization')
```

Comment:
I'd perhaps prefer sending back all headers. We'll do that in the next PR that needs to edit this.